src/Microsoft.Azure.WebJobs.Host/Executors/FunctionExecutor.cs

```diff
@@ -552,9 +576,119 @@ private static ITaskSeriesTimer StartParameterLogTimer(IRecurrentCommand updateC
+            Dictionary<string, object> properties = new Dictionary<string, object>();
+
+            Task<object> invokeTask = null;
+            try
+            {
+                await InvokeExecutingFilters(filters, parameterHelper, properties, functionCancellationTokenSource, instance, logger, jobHost, config);
+                invokeTask = invoker.InvokeAsync(invokeParameters);
+                functionResult = new FunctionResult(true);
+            }
+            catch (Exception ex)
+            {
+                invokeException = ex;
+                functionResult = new FunctionResult(ex);
+                throw;
+            }
+            finally
+            {
+                await InvokeExecutedFilters(filters, parameterHelper, properties, functionCancellationTokenSource, instance, logger, jobHost, config, functionResult);
+                if (invokeException != null)
+                {
+                    throw invokeException;
+                }
+            }
+
+            return invokeTask;
+        }
+
+
+        private static List<IFunctionInvocationFilter> GetFilters(FunctionDescriptor functionDescriptor, object instance)
+        {
+            List<IFunctionInvocationFilter> filters = new List<IFunctionInvocationFilter>();
+
+            if (instance != null)
+            {
+                IFunctionInvocationFilter instanceFilter = instance as IFunctionInvocationFilter;
+
+                if (instanceFilter != null)
+                {
+                    filters.Add(instanceFilter);
+                }
+            }
+            
+            // Check for method filters
+            if (functionDescriptor != null)
+            {
+                if (functionDescriptor.FilterAttributes != null)
+                {
+                    filters.AddRange(functionDescriptor.FilterAttributes);
+                }
+            }
+
+            // Check for class filters
+            if (functionDescriptor != null)
+            {
+                if (functionDescriptor.DeclaringFilterAttributes != null)
+                {
+                    filters.AddRange(functionDescriptor.DeclaringFilterAttributes);
+                }
+            }
+
+            return filters;
+        }
+
+        private static async Task InvokeExecutingFilters(List<IFunctionInvocationFilter> filters, ParameterHelper parameterHelper,
+            Dictionary<string, object> properties, CancellationTokenSource functionCancellationTokenSource, 
+            IFunctionInstance instance, ILogger logger, JobHost jobHost, JobHostConfiguration config)
+        {
+            if (filters.Any())
+            {
+                // Create the context
+                FunctionExecutingContext context = new FunctionExecutingContext(instance.Id, instance.FunctionDescriptor.FullName, parameterHelper.GetParametersAsDictionary(), properties, logger);
+                context.JobHost = jobHost;
```

Comment:
no, they're internal properties that aren't publicly setable. 

---
In reply to: [128639757](https://github.com/Azure/azure-webjobs-sdk/pull/1244#discussion_r128639757) [](ancestors = 128639757)